### PR TITLE
Restrict widget size

### LINKED ISSUE #62: kernel dies if widget is resized to a very large height, this sometimes happens unintentionally
This is a bizarre issue that seems to happen if the widget is left unattended for a long period of time (~30+ minutes?) . If you then try to resize it after this long unattended period it elongates vertically downward extremely fast, as if I suddenly increased my mouse cursor speed by an order of magnitude. The kernel then crashes. 

I'll try to reproduce and provide more info soon.

### PR DESCRIPTION
**Pull request by almarklein**
Closes #62

### CODE REVIEW

**Comment by kushalkolar** [comment]
Is there a way to make this settable or based on the browser window width/height? We often use 4k screens and have large canvases.

File: js/lib/widget.js [hunk 1]
```diff
@@ -316,6 +316,9 @@ export class RemoteFrameBufferView extends DOMWidgetView {
         if (!this.el.style.width && this.model.get('css_width')) {
             this.el.style.width = this.model.get('css_width');
             this.el.style.height = this.model.get('css_height');
+            // prevent massive size due to auto-scroll (issue #62)
+            this.el.style.maxWidth = "1536x";
+            this.el.style.maxHeight = "1024px";
```
Comment (almarklein) [inline js/lib/widget.js:321]: Allow at least 1024x1024, or larger, based on the size of the browser window at the time the canvas is created.
```suggestion
            this.el.style.maxWidth = Math.max(1024, window.innerWidth) + 'px';
            this.el.style.maxHeight = Math.max(1024, window.innerHeight) + 'px';
```

**Comment by almarklein** [comment]
Good point! I update the pr.

**Comment by almarklein** [comment]
Looks like we need to drop Python 3.8. I'm on holiday now, so won't do it now. Though I would merge a PR that fixes that 😉. Otherwise I can fix it later.

**Pull request merged**